site/ja/tutorials/load_data/images.ipynb

```diff
@@ -0,0 +1,3254 @@
+{
```

Comment:
<p>memo: This cell was changed in r2 notebook. Since path_ds does not have output_shapes property.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Yes.</p>